src/window.js

```diff
@@ -24,7 +24,34 @@ var ToolboxWindowBase = GObject.registerClass({
         this.hack_toolbox_skeleton = this._createHackToolboxSkeletonOnPath(objectPath);
         this.hackable_proxy = this._maybeGetHackableProxySync();
 
-        this.application.enableFlipBack = false;
+        this._flipBack = new Gio.SimpleAction({
+            name: 'flip-back',
+            parameter_type: new GLib.VariantType('(ss)'),
+        });
+        this._flipBack.connect('activate', this._onFlipBack.bind(this));
+    }
+
+    _onFlipBack(action, parameterVariant) {
+        this.application.hold();
```

Comment:
I think we don't need to hold/release the application now, since the window will keep the application alive.